tests/test_cluster.nim

```diff
@@ -117,6 +117,26 @@ suite "cluster suite":
     check b.left == 3
     check b.right == 4
 
+  test "test bounds: filter out soft-clips that are inconsistent with the center mass":
+    var reads = @[
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 100, split: Soft.right),
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 123, split: Soft.none),
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 223, split: Soft.none),
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 223, split: Soft.none),
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 223, split: Soft.none),
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 253, split: Soft.none),
+      tread(tid:1'i32, repeat: ['A', 'T', 'G', '\x00', '\x00', '\x00'], position: 283, split: Soft.none),
+      ]
+
+    let max_clip_dist:uint16 = 50
+    var cl = Cluster(reads:reads)
+    var b = cl.bounds(max_clip_dist)
+    check b.left == 223
+    check b.right == 224
+    check b.left_most == 100
```

Comment:
should it drop (or flag) the read(s) that don't fit with the event?

Reply:
Yes, that would be better. Because then we could update the center mass and the start/end of the event accordingly.

Reply:
@brentp what are you thinking about this? Do you think it's worth changing the implementation to completely drop those reads rather than just excluding them from the left/right bounds calculation?